### PR TITLE
ran babel-upgrade to keep up with beta

### PR DESCRIPTION
**Pull request by collin**
We had a stackathon student run into deprecation warnings for the `stage-*` babel presets.

Looks like those preset bundles are on the way out in favor of explicit opt-in.

I ran the codemod from babel: https://github.com/babel/babel-upgrade

And this is the result.

### CODE REVIEW

**Comment by glebec** [comment]
Closes #126

File: .babelrc [hunk 1]
```diff
@@ -1,12 +1,31 @@
 {
-  presets: ['@babel/react', '@babel/env', ["@babel/preset-stage-2", { "decoratorsLegacy": true }]]
-  /*
+  "presets": [
+    "@babel/react",
+    "@babel/env"
+    /*
     Babel uses these "presets" to know how to transpile your Javascript code. Here's what we're saying with these:
-
```
Comment (glebec) [inline .babelrc]: is there a reason for losing this line break? It seems arbitrary.

Comment (collin) [reply .babelrc]: No, just a copy/paste issue when putting the .babelrc back together.

**Comment by collin** [comment]
I am ambivalent re: the other package updates.

In theory they are all minor upgrades, but that's no guarantee.

Our micro test-suite passes... so do we feel good about that?

( This is a potential argument a blessed grace-shopper solution with tests that can stand as a QA tool for boilermaker upgrades. )

**Comment by glebec** [comment]
I feel similarly. Just wanted to bring it up as a sanity check.

**Pull request merged**